tests/face_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import rhino3dm
+from pathlib import Path
+from honeybee.face import Face
+from honeybee_3dm.face import to_face
+
+
+def test_room():
+    path = str((Path.cwd()).joinpath('tests', 'test.3dm'))
+    room = to_face(path)[0]
+    assert isinstance(room, Face)
```

Comment:
You probably want more checks to make sure it has been translated correctly. Right now you are just checking that it has been translated.